### DISCUSSION TITLE
indication on tray

### DISCUSSION THREAD

**Discussion by mcdemon05** [davidsmorais/kuro#107] [Help]
I decided to change my home from Windows to Linux.
But I was faced with a choice, use kuro or go to another task sheet.

I decided to try kuro, it seems to work ... But I'm not satisfied that the kuro icon in the tray or on the application bar does not show an indication of the number of uncompleted tasks assigned for today.
For example in Windows 10, the to-do app shows the number of uncompleted tasks. Similarly, for example, the number of unread messages in Telegram.

And it turns out now I can just forget that I didn’t do some task and nothing will remind me of it.
While on Windows, opposite the Microsoft To-Do icon was a number with the number of uncompleted tasks for today.

is it possible to somehow use this functionality?
Or will I still have to switch to another product?

P.s. I use Fedora with GNOME

**Comment by davidsmorais** [comment]
Hey @mcdemon05 , thanks for the message.

The 'daily todos' information lives on the server side, Microsoft's servers, therefore it's not possible to add this functionality to Kuro. You have to open it and check your daily tasks manually.

Seems you'll have to switch to another product :cry:

**Comment by mcdemon05** [comment]
But can't kuro just see the presence of outstanding or overdue tasks for today?
Not even the number, but simply the very fact of their presence.
And based on this, change the tray icon.
Maybe via api?

**Reply by mcdemon05** [reply]
https://learn.microsoft.com/graph/api/resources/todo-overview?view=graph-rest-1.0

**Comment by davidsmorais** [comment]
@mcdemon05, thanks for the suggestion, but Kuro isn't tapping into Microsoft's ToDo API, it's really just a wrapper around the web client for Linux
